### PR TITLE
Update labeling and exchange rate provider for Envoy

### PR DESCRIPTION
**Pull request by sethforprivacy**
Labeling was added in 1.4.0 alongside coin control, and we host our own exchange rate provider server.

### CODE REVIEW

**Review by maxirosson — APPROVED** [review]

**Comment by maxirosson** [comment]
Thanks for the collaboration :)

**Comment by maxirosson** [comment]
There is an issue with the JSON format

**Comment by maxirosson** [comment]
You can run

```
cd scripts/
node json-format.js
```

after fixing the issue, to verify the json is well formatted.

File: items/envoy.json [hunk 1]
```diff
@@ -341,9 +341,15 @@
       "flag": "negative"
     },
     "transaction-labeling": {
-      "value": "NO",
-      "supported": false,
-      "flag": "negative"
+      "value": "YES",
+      "supported": true,
+      "flag": "positive"
```
Comment (maxirosson) [inline items/envoy.json:346]: I missing `,` here

Comment (sethforprivacy) [reply items/envoy.json]: Fixed, thanks for that! Was too late at night to be making PRs, clearly :sweat_smile:

**Comment by maxirosson** [comment]
Not sure why it is still failing. I will merge it and then fix it locally. Thanks

**Pull request merged**